### DISCUSSION TITLE
Does a speaker always need to be specified when using voice assist?

### DISCUSSION THREAD

**Discussion by mph070770** [music-assistant/support#3101] [Q&A] [answered]
If the voice satellite and playback speaker both have a HA location as "kitchen", is it necessary to say:

"Play Queen on the kitchen speaker"

or is it possible to just say:

"Play queen"

and the default speaker for the room where the microphone is located is used instead?

**Comment by OzGav** [comment, marked as answer]
No that is not currently possible. You can see what intents are possible by using the HA DEVELOPER TOOLS >> ASSIST
